v2/api/network/v1beta20201101storage/load_balancer_types_gen.go

```diff
@@ -1358,8 +1359,13 @@ func (embedded *BackendAddressPool_STATUS_LoadBalancer_SubResourceEmbedded) Assi
 			if err != nil {
 				return errors.Wrap(err, "calling AssignProperties_From_SubResource_STATUS() to populate field SubResource_STATUSStash from LoadBalancingRules")
 			}
+			var subResourceSTATUSStashLocal v1api20180501s.SubResource_STATUS
+			err = subResourceSTATUSStashLocal.AssignProperties_From_SubResource_STATUS(&subResourceSTATUSStash)
+			if err != nil {
```

Comment:
@theunrepentantgeek just confirming this is right. I think it is given the addition of the older API version but just confirming.

Reply:
Yes it is correct. All `beta` versions come before all `api` versions, so the sequence is this:

... -> `v1beta20201101storage` -> `v1api20180501storage` -> `v1api20200601storage` -> `v1api20201101storage`

The extra code is to handle the conversion required by introducing `v1api20180501storage` into the midst of the conversion graph.